server/user-topics/user-topics.controller.js

```diff
@@ -0,0 +1,56 @@
+const UserTopic = require('./user-topics.model');
+
+
+module.exports = {
+  addUserTopic,
+  getAllUserTopics,
+  RemoveUserTopic,
+};
+/******** PUBLIC ********/
+
+function addUserTopic(req, res) {
+  // check to see if the user already has this topic.
+  // if they do there is no need to add it to the db
+  UserTopic.find({
+    where: {
+      topicId: req.topic.topicId,
+      userId: req.topic.userId,
+    }
+  })
+  .then(row => {
+    if (!row) {
+      UserTopic.create({
+        topicId: req.topic.topicId,
+        userId: req.topic.userId,
+      })
```

Comment:
in the then res.send something since this is ifnal middleware (res.sendStatus(201)